index.html

```diff
@@ -1,140 +1,170 @@
+	<!-- Custom Stylesheet -->
+	<link rel="shortcut icon" href="favicon.ico">
+	<link rel="stylesheet" href="css/custom.css">
+	<!--[if lt IE 9]>
 		<script src="js/html5shiv.js"></script>
 		<script src="js/respond.js"></script>
 		<![endif]-->
-	</head>
-	<body>
-		<header>
-			<nav class="content">
-				<a href="index.html"><h1 class="gittext lowercase">Git<span>for Windows</span></h1></a>
-				<div class="version"><a href="https://github.com/git-for-windows/git/releases/tag/v2.34.0.windows.1" title="Version 2.34.0 was published on Mon, 15 Nov 2021 19:52:38 UTC">Version 2.34.0</a></div>
-				<ul class="list-unstyled">
-					<li><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ" target="_blank">FAQ</a></li>
-					<li><a href="https://github.com/git-for-windows/git" target="_blank">Repository</a></li>
-					<li><a href="https://groups.google.com/group/git-for-windows" target="_blank">Mailing List</a></li>
-				</ul>
-			</nav>
-			<section class="content feature">
-				<div class="imagerollover">
-					<img src="img/gwindows_logo.png" alt="Git Logo">
-					<img class="top" src="img/git_logo.png" alt="Git Logo">
+  </head>
+
+  <body>
+	<!-- navbar -->
+	<nav class="navbar navbar-light navbar-expand-lg fixed-top shadow-sm bg-white">
+		<a href="index.html" class="navbar-brand">
+			<h1 class="gittext lowercase">
+				Git<span>for Windows</span>
+			</h1>
+		</a>
+		<button type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+			aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"
+			class="navbar-toggler"><span class="navbar-toggler-icon"></span></button>
+		<div id="navbarSupportedContent" class="collapse navbar-collapse">
+			<ul class="navbar-nav ml-auto">
+				<li class="nav-item"><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ"
+						target="_blank" class="nav-link">FAQ</a></li>
+				<li class="nav-item"><a href="https://github.com/git-for-windows/git" target="_blank"
+						class="nav-link">Repository</a></li>
+				<li class="nav-item"><a href="https://groups.google.com/group/git-for-windows" target="_blank"
+						class="nav-link">Mailing List</a></li>
+				<li class="nav-item"><a href="https://gitter.im/git-for-windows/git" target="_blank"
+						class="nav-link">Gitter</a></li>
+				<li class="nav-item"><a href="https://groups.google.com/g/git-mentoring" target="_blank"
+						class="nav-link">Git Mentoring</a></li>
+			</ul>
+		</div>
+	</nav>
+	<section class="bg-light">
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-6 order-2 order-lg-1">
+					<img src="img/gwindows_logo.png" alt="Git Logo" class="img-fluid">
 				</div>
-				<article>
-					<div class="vcentercontainer">
-						<div>
+				<div class="col-md-6 col-md-offset-3 col-md-push-3">
+					<div class="text-right">
+						<p class="lead">
 							<h2>We bring the awesome <span class="gittext">Git</span> SCM to Windows</h2>
```

Comment:
Since we're going to the trouble of making a change, could we consider dropping `SCM` from the heading? Any of the following would be preferable:
- We bring Git to Windows
- We bring the power of Git to Windows
- Git + Windows = awesome

I know it's baked into the upstream Git domain name, but in my experience, few people use this term anymore. The Git homepage itself uses it only in a derisive way (emphasis mine):

> Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.
>
> Git is easy to learn and has a tiny footprint with lightning fast performance. **It outclasses SCM tools** like Subversion, CVS, Perforce, and ClearCase with features like cheap local branching, convenient staging areas, and multiple workflows.